_includes/partner-script.js

```diff
@@ -1344,13 +1351,104 @@
 			link: 'http://harmonycloud.cn/products/rongqiyun/',
 			blurb: 'Harmonycloud - Harmonycloud Container Platform'
 				},
+		{
+			type: 3,
+			name: 'Woqutech',
+			logo: 'woqutech',
+			link: 'http://www.woqutech.com/product/product-16-247.html#sss',
+			blurb: 'Woqutech - QFusion'
+				},
+		{
+			type: 3,
+			name: 'Baidu',
+			logo: 'baidu',
+			link: 'https://cloud.baidu.com/product/cce.html',
+			blurb: 'Baidu Cloud - Baidu Cloud Container Engine'
+				},
 		{
 			type: 3,
 			name: 'ZTE',
 			logo: 'zte',
 			link: 'https://sdnfv.zte.com.cn/zh-CN/home',
```

Comment:
As a ZTEr, I'd like here to use this link https://sdnfv.zte.com.cn/en/home for i18n :)

Reply:
Updated - thanks!